docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -49,9 +49,15 @@ working:
 Now we are able to send and receive messages through the broker (by default
 mosquitto uses port 1883). 
 
-This tutorial uses python scripts so we need to install one python library:
+This tutorial uses python scripts so we need to install the python library paho-mqtt.
+You need 'pip3' to install this module, so if you have not already done so, you will
+need to install 'pip3':
 
-    sudo easy_install paho-mqtt
+  sudo apt-get install python3-pip
+
+ Now you can install paho-mqtt:
+ 
+  sudo pip3 install paho-mqtt
 
 All scripts were tested successully on a EV3 running the latest ev3dev version
 (as of 12 May 2016) and also on a Raspberry Pi 3 with a BrickPi running the same
```

Comment:
Ah, good catch on the shebangs! @KWSmit Can you update the date and surrounding text that I commented on and modify the shebangs in the example scripts?